test/wdio/external-imports/cmp.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { Fragment, h } from '@stencil/core';
+import { render } from '@wdio/browser-runner/stencil';
+
+describe('external-imports', () => {
+  beforeEach(async () => {
+    render({
+      template: () => (
+        <>
+          <external-import-a></external-import-a>
+          <external-import-b></external-import-b>
+          <external-import-c></external-import-c>
+        </>
+      ),
+    });
+  });
+  it('render all components without errors', async () => {
+    let elm = $('external-import-a');
```

Comment:
Non-blocking - this can be `const` now
```suggestion
    const elm = $('external-import-a');
```